src/app/AppContainer.js

```diff
@@ -12,25 +12,14 @@ import type { BlockType } from '../store/types';
 
 type PropsType = {
 	rootClientId: ?string,
-	isBlockSelected: string => boolean,
 	initialHtmlModeEnabled: boolean,
 	showHtml: boolean,
 	editedPostContent: string,
-	selectedBlockIndex: number,
-	blocks: Array<BlockType>,
-	onInsertBlock: ( BlockType, number, ?string ) => mixed,
-	onMerge: ( string, string ) => mixed,
-	onMoveDown: ( string, ?string ) => mixed,
-	onMoveUp: ( string, ?string ) => mixed,
-	onRemove: ( string, ?string ) => mixed,
-	onToggleBlockMode: ?string => mixed,
-	onResetBlocks: Array<BlockType> => mixed,
-	onSelect: string => mixed,
-	clearSelectedBlock: void => void,
-	onAttributesUpdate: ( string, mixed ) => mixed,
 	initialHtml: string,
+	resetBlocks: Array<BlockType> => mixed,
 	setupEditor: ( mixed, ?mixed ) => mixed,
-	clientId: string,
+	toggleBlockMode: ?string => mixed,
+	getBlocks: () => Array<BlockType>,
```

Comment:
WDYT about adding the `post` property to the PropsType definition? We are optionally expecting it in line 31 even though Flow doesn't complain (I guess because of the `||`ing).